modules/jre/run/bash/module.yaml

```diff
@@ -0,0 +1,17 @@
+schema_version: 1
+name: jboss.container.java.jre.run.bash
```

Comment:
Please rename the module to remove the ".bash" suffix, and move the files accordingly (from `modules/jre/run/bash` to `modules/jre/run`)